config/node-schedule.js

```diff
@@ -0,0 +1,40 @@
+const schedule = require("node-schedule");
+const Pin = require("../models/Pin");
+const getDate = require("../utils/getDate");
+
+function nodeSchedule() {
+  const rule = new schedule.RecurrenceRule();
+
+  rule.second = 0;
+
+  async function updatePin() {
+    const pins = await Pin.find({ active: true }).lean();
+
+    for (let i = 0; i < pins.length; i++) {
+      const pin = pins[i];
+      const { createdAt, savedAt, _id: id } = pin;
+
+      if (savedAt) {
+        const timeOver = getDate(savedAt);
+
+        if (timeOver) {
+          await Pin.findByIdAndUpdate(id, { active: false });
+        }
+
+        return;
+      }
+
+      const timeOver = getDate(createdAt);
+
+      if (timeOver) {
+        await Pin.findByIdAndUpdate(id, { active: false });
+      }
+    }
+  }
+
+  const job = schedule.scheduleJob("*/1 * * * * *", () => {
+    updatePin();
+  });
```

Comment:
제가 잘 몰라서 그러는데 "*/1 * * * * *" 는 어떤 의미일까요?
그리고 아래와 같이 바꿔줄 수도 있을 것 같습니다!
```suggestion
  const job = schedule.scheduleJob("*/1 * * * * *", () => updatePin());
```

Reply:
node schedule에서 별모양의 위치에 따라 값을 주면 스케줄을 실행할 설정시간을 지정할수 있습니다

[node-schedule](https://www.npmjs.com/package/node-schedule)